scripts/checkout.js

```diff
@@ -0,0 +1,92 @@
+import { cart } from "../data/cart.js";
+import { products } from "../data/products.js";
+
+let cartSummaryHTML = "";
+
+cart.forEach((cartItem) => {
+  const productId = cartItem.productId;
+
+  let matchingProduct;
+
+  products.forEach((product) => {
+    if (product.id === productId) {
+      matchingProduct = product;
+    }
+  });
+
+  cartSummaryHTML += `
+  <div class="cart-item-container">
+        <div class="delivery-date">Delivery date: Tuesday, June 21</div>
+
+        <div class="cart-item-details-grid">
+        <img
+            class="product-image"
+            src="${matchingProduct.image}"
+        />
+
+        <div class="cart-item-details">
+            <div class="product-name">
+            ${matchingProduct.name}
+            </div>
+            <div class="product-price">
+            $${matchingProduct.priceCents / 100}</div>
+            <div class="product-quantity">
```

Comment:
this is a normalization

Reply:
using money.js utils we can use this function at multiple places
```suggestion
            $${formatCurrency(matchingProduct.priceCents)}</div>
```